enclave/src/error.rs

```diff
@@ -1,8 +1,7 @@
 use crate::rpc;
 use derive_more::{Display, From};
 use sgx_types::sgx_status_t;
-
-use std::result::Result as StdResult;
+use std::{prelude::v1::Box, result::Result as StdResult};
```

Comment:
In other places I think we use the `Box` from the `alloc` crate - what's the difference and which one should be prefer? (`alloc` crate of `std::prelude::v1` ?)

Reply:
This does not matter, it is the same. `std` re-exports the one form `alloc`.

Reply:
That I assumed/hoped for 😄 and for style consistency, which import should we use in you opinion?

Reply:
I would take it from std if available. Due to some internal rust limitations, you'd need to add `extern crate alloc` otherwise. (I think this is tried to be resolved)